rs/nervous_system/integration_tests/src/lib.rs

```diff
@@ -13,3 +13,27 @@
 
 pub mod create_service_nervous_system_builder;
 pub mod pocket_ic_helpers;
+
+use std::time::Instant;
+
+pub struct Section {
```

Comment:
I like this idea - very cool.

Can we name this something that indicates what it does?  BlockTimeLogger?  TestSectionTimer?  My name game isn't amazing this morning, but "Section" is too generic IMO.

Reply:
I'll go with `SectionTimer`